lib/webpacker/routes/railtie.rb

```diff
@@ -7,8 +7,8 @@ class Engine < ::Rails::Engine
       config.webpacker.routes.default_url_options = {}
       config.webpacker.routes.camel_case = false
 
-      config.after_initialize do |app|
-        generate = -> { Webpacker::Routes.generate(app.tap(&:reload_routes!)) }
+      initializer 'webpacker.routes', :after => :set_routes_reloader_hook do |app|
```

Comment:
I also tried this in my other PR. I've got no idea why, but in my project this causes the sprockets assets loading to fail with `Sprockets::Rails::Helper::AssetNotFound`. The previous `config.after_initialize` works fine though.

Reply:
That is surprising. Do you have a stack trace? Do you remember if it happened during initialization or during reload? Because I think it would work similarly to use `config.after_initialize` and `app.reloader.to_run(:after, &generate)`

Reply:
The application starts ok, but for example when trying to render a page which has a `<%= stylesheet_link_tag 'application' %>` then it fails. Using rails 5.24 and sprockets 4.0.0.

Reply:
I was able to reproduce that as well. I'm not sure why, but using `:after => :set_routes_reloader_hook` causes the sprockets initializer to move too. 44da8116deda9d9d97637198bfd2742c2a44aa31 uses `after_initialize` again and should fix it.